### DISCUSSION TITLE
Boya BY-LM40 mic

### DISCUSSION THREAD

**Discussion by iain-uk** [mcguirepr89/BirdNET-Pi#892] [General]
Boya BY-LM40 mic
New to birding and setup a new station using a pi4 4gb and a Boya BY-LM40 mic
https://www.amazon.co.uk/gp/product/B09N6YR9LM 
Impressed with results and shows as native device 

![image](https://user-images.githubusercontent.com/51044235/236633734-3ecf09d9-79be-49a4-9960-925c2a065d66.png)

it’s an active USB mic and requires usb 5v power to work correctly 
it has a 4M lead and currently just dangling out of window 

![image](https://user-images.githubusercontent.com/51044235/236633769-a50189bf-71aa-4578-be12-59c40d8f2324.png)

 
Going to attempt to build a parabolic dish out of a salad bowl and plastic bottle cut down to shield mic from rain, need to find the focal point of dish for mic position first with a mobile phone running a constant tone via a frequency generator set up about 5M away and note mic levels on pc

![image](https://user-images.githubusercontent.com/51044235/236633787-29f71900-960f-4c63-82e6-3a968d9a5b1a.png)

 
Hopefully will eliminate unwanted signals from neighbours cars, dogs, kids, lawnmowers, airplanes, motorway (1/2 mile away) etc. and angle it  horizontal with a view of catching more birds on the wing and migratory

**Comment by becausebirds** [comment]
So awesome!

FWIW, my lav mic has been in many rainstorms and keeps on chugging! My box is installed on the back of my house in an enclosure and the microphone is just plugged into it and hangs there haha.

**Comment by rgbdeluxe** [comment]
@iain-uk Hey, you find the focal point with this formular. https://www.analyzemath.com/parabola/focus-of-parabola-calculator.html
Just type in the measurement of your salad bowl and it will calculate it for you. I used this for my setUp. Works quite well. 
Still a lot of backgorund noise but in gerneral, I get better recordings
